OJP_JourneySupport.xsd

```diff
@@ -23,22 +23,22 @@
 	</xs:complexType>
 	<xs:complexType name="ProductCategoryStructure">
 		<xs:annotation>
-			<xs:documentation>Product category based on NeTEx</xs:documentation>
+			<xs:documentation>Product category based on NeTEx. A product category is a classification to particular VEHICLE JOURNEYs to express some common properties of journeys form marketing and fare products. </xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
 			<xs:element name="Name" type="InternationalTextStructure" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Name of the product category.</xs:documentation>
+					<xs:documentation>Name of the product category. This is the full name for this product category. E.g. "Autoreisezug" in Switzerland or "Dampfschiff"</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="ShortName" type="InternationalTextStructure" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Short name or acronym of product category.</xs:documentation>
+					<xs:documentation>Short name or acronym of product category. If you intend to publish the product category, this is were you would put, the information that you want to associate with the journey. E.g. "ARZ" in Switzerland or "BAV".</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element ref="ProductCategoryRef" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>A reference to the product category.</xs:documentation>
+					<xs:documentation>A reference to the product category. This is the internal code that is used within the reference frameworks (NeTEx, Siri). It is usually not displayed, but interpreted by the technical system. E.g. ch:1:TypeOfProductCategory:ARZ (Autoreisezug in Switzerland) or ch:1:TypeOfProductCategory:BAV. </xs:documentation>
```

Comment:
`This is the internal code used within the reference frameworks (NeTEx, SIRI)`

Reply:
changed.